### PR TITLE
Feature/lesson13

### PR DESCRIPTION
**Pull request by AndreiChernukho**

### CODE REVIEW

File: src/com/tms/lesson13/service/PeopleServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+import com.tms.lesson13.entity.People;
+
+import java.sql.*;
+import java.util.ArrayList;
+
+public class PeopleServiceImpl {
+    private Connection connection;
+
+    public PeopleServiceImpl(Connection connection) {
```
Comment (gr00shik) [inline src/com/tms/lesson13/service/PeopleServiceImpl.java:11]: as getConnection is a static you can get it directly

File: src/com/tms/lesson13/service/PeopleServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+            Date birthday = resultSet.getDate(4);
+            return id + ", " + name + ", " + sex + ", " + birthday;
+        } catch (SQLException exc) {
+            return "Name " + name +" not found";
+        }
+    }
+
+    public ArrayList<People> getAllPeople() {
+
+        ArrayList<People> people = new ArrayList<People>();
```
Comment (gr00shik) [inline src/com/tms/lesson13/service/PeopleServiceImpl.java:69]: List = new ArrayList

**Review by gr00shik — COMMENTED** [review]
ok

File: src/com/tms/lesson13/service/PeopleServiceImpl.java (hunk 1 above)
Comment (AndreiChernukho) [reply src/com/tms/lesson13/service/PeopleServiceImpl.java]: fix

File: src/com/tms/lesson13/service/PeopleServiceImpl.java (hunk 2 above)
Comment (AndreiChernukho) [reply src/com/tms/lesson13/service/PeopleServiceImpl.java]: fix

**Pull request merged**